### PR TITLE
NXP-22989: moves document edit to dialog

### PR DESCRIPTION
**Pull request by pncosta**

### CODE REVIEW

File: elements/nuxeo-document-actions/nuxeo-document-edit-action.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <nuxeo-document-edit id="dialog" document="[[document]]"></nuxeo-document-edit>
+
+  </template>
+  <script>
+    Polymer({
+      is: 'nuxeo-document-edit-action',
+      behaviors: [Nuxeo.I18nBehavior, Nuxeo.FiltersBehavior],
+      properties: {
+
+        /**git
```
Comment (adilioaraujo) [inline elements/nuxeo-document-actions/nuxeo-document-edit-action.html]: oops :)

File: elements/nuxeo-web-ui-bundle.html [hunk 2]
```diff
@@ -233,6 +233,12 @@
   </template>
 </nuxeo-slot-content>
 
+<nuxeo-slot-content name="clipboardToggleDocumentAction" slot="DOCUMENT_ACTIONS" order="100">
```
Comment (adilioaraujo) [inline elements/nuxeo-web-ui-bundle.html]: Change `name="clipboardToggleDocumentAction"` to `name="editDocumentAction"`

**Comment by adilioaraujo** [comment]
:+1:

File: elements/document/nuxeo-document-edit.html [hunk 3]
```diff
@@ -136,11 +122,16 @@
           properties: this._dirtyProperties
         };
 
-        this.$.doc.put().then(this._refresh.bind(this));
+        this.$.doc.put().then(
```
Comment (afjusto) [inline elements/document/nuxeo-document-edit.html]: nit: you could move the `function` to the same line as `then`, then you can remove the unnecessary indentation and there's a space missing between `function()` and the `{`.

(auto-format ftw ;))

File: elements/document/nuxeo-document-edit.html [hunk 4]
```diff
@@ -73,13 +51,17 @@
 
     <nuxeo-document id="doc" doc-id="[[document.uid]]" response="{{document}}" headers="[[headers]]"></nuxeo-document>
 
-    <form id="form" is="iron-form">
-      <nuxeo-document-layout id="document-edit" document="{{document}}" layout="edit"></nuxeo-document-layout>
-      <div class="actions">
+    <nuxeo-dialog id="dialog" with-backdrop>
```
Comment (Gabez0r) [inline elements/document/nuxeo-document-edit.html]: add missing import `nuxeo-dialog`

File: elements/document/nuxeo-document-edit.html [hunk 5]
```diff
@@ -73,13 +51,17 @@
...
+      <paper-dialog-scrollable>
```
Comment (Gabez0r) [inline elements/document/nuxeo-document-edit.html]: add missing import `paper-dialog-scrollable`

**Pull request merged**